docs/setup/kubeedge_precheck.md

```diff
@@ -0,0 +1,81 @@
+# KubeEdge Pre-Check
+
+## Status Check
+
+After the Cloud and Edge parts have started, you can use below command to check the edge node status.
+
+On cloud host run,
+
+```
+kubectl get nodes
+
+NAME         STATUS     ROLES    AGE     VERSION
+testing123   Ready      <none>   6s      0.3.0-beta.0
+```
+
+Please make sure the status of edge node you created is **ready**.
+
+## Deploy Application on cloud side
+
+Try out a sample application deployment by following below steps.
+
+```shell
+kubectl apply -f $GOPATH/src/github.com/kubeedge/kubeedge/build/deployment.yaml
+deployment.apps/nginx-deployment created
+```
+
+**Note:** Currently, for applications running on edge nodes, we don't support `kubectl logs` and `kubectl exec` commands(will support in future release), support pod to pod communication running on **edge nodes in same subnet** using edgemesh.
+
+Then you can use below command to check if the application is normally running.
+
+Check the pod is up and is running state
+
+```
+kubectl get pods
+NAME                               READY   STATUS    RESTARTS   AGE
+nginx-deployment-d86dfb797-scfzz   1/1     Running   0          44s
+```
+
+Check the deployment is up and is running state
```

Comment:
Same as above. But here I think it is not showing a status? Please check.

Reply:
@anvithks Did you meant that command below is not showing a status?
```
kubectl get deployments

NAME               READY   UP-TO-DATE   AVAILABLE   AGE
nginx-deployment   1/1     1            1           63s
```